students/msemerni/hw-6+7/js/task-3.js

```diff
@@ -0,0 +1,52 @@
+//// Task 3
+//// Transform each element in the initial array with numbers
+//// that represent days of the week to an array with human readable day names
+
+//// first
+const numberDays = [3, 4, 5, 1, 6, 2, 7];
+
+function convertNumberDays1(daysArray) {
+  let humanDays = [];
+  daysArray.forEach(day => {
+    if (day === 1) {
```

Comment:
`switch` will definitely looks better here

Reply:
I've never heard that. Could you share a links?

Reply:
https://www.google.com/search?q=don%27t+use+switch+statement+javascript&oq=dont+use+switch+&aqs=chrome.3.69i57j0i19l3j0i10i19i22i30l3j0i19i22i30j0i10i19i22i30.16514j0j4&client=ms-android-xiaomi-rev1&sourceid=chrome-mobile&ie=UTF-8#ip=1

Reply:
Yeap, but there are nothing about `never` using of `switch`. 
Definitely, there could be scenarios when it's better to replace `switch` to another constructions, but **in general it's absolutely OK to use it in production**.